go/externals/proof_support_facebook.go

```diff
@@ -159,7 +159,12 @@ func (t FacebookServiceType) NormalizeUsername(s string) (string, error) {
 func (t FacebookServiceType) NormalizeRemoteName(ctx libkb.ProofContext, s string) (string, error) {
```

Comment:
I took a look and I don't think we call `NormalizeName` directly -- we just use `libkb.CheckUsername.F`, `libkb.NewNormalizedUsername`, `externals.NormalizeSocialAssertion`, `externals.AssertionParseAndOnly`.